codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -637,7 +646,7 @@ private List<AbiDefinition.NamedType> extractStructs(
                             parameters.addAll(definition.getOutputs());
                             return parameters.stream()
                                     .map(this::normalizeNamedType)
-                                    .filter(namedType -> namedType.getType().startsWith("tuple"));
```

Comment:
This was added in my previous PR merged https://github.com/hyperledger/web3j/pull/2061/files#diff-11944d4d901c6c86e0bb4f4bb1af18f9b3644a325a39e7cc7437878613330294R590

However, after fixing `normalizeNamedType` to handle static array, this is no longer needed